### ISSUE TITLE
Why is reward model training not logged to W&B?

### ISSUE THREAD

**Issue by RylanSchaeffer** [tlc4418/llm_optimization#6]
I'm now training reward models using your code and I discovered that reward models are not logged during their creation.

https://github.com/tlc4418/llm_optimization/blob/main/configs/config_rm.yaml#L37

Why is this?

**Comment by tlc4418** [comment]
Yeah I think during the refactor I removed this to simplify it for new users (so they can set up their own logging or log the way they want. 

Feel free to set this back to true and set a wandb_entity in the RM training [here](https://github.com/tlc4418/llm_optimization/blob/main/src/reward_modeling/training/trainer_rm.py#L146). I think you will also need to add the following lines of code [before creating the Trainer](https://github.com/tlc4418/llm_optimization/blob/main/src/reward_modeling/training/trainer_rm.py#L317):
```
wandb.init(
    project="reward-model",
    entity=training_conf.wandb_entity,
    resume=training_conf.resume_from_checkpoint,
    name=f"{training_conf.model_name}-{training_conf.log_dir}-rm",
    config=training_conf,
)
```
